vars/githubCreatePullRequest.groovy

```diff
@@ -30,8 +30,10 @@ def call(Map args = [:]) {
   }
   def titleValue = args.containsKey('title') ? args.title : error('githubCreatePullRequest: title parameter is required.')
   def descriptionValue = args.get('description', '')
-  def assign = args.containsKey('assign') ? "--assign ${args.assign}" : ''
-  def reviewer = args.containsKey('reviewer') ? "--reviewer ${args.reviewer}" : ''
+  def assign = args.get('assign', '')
+  assign = (assign.trim()) ? "--assign ${args.assign}" : ''
+  def reviewer = args.get('reviewer', '')
+  reviewer = (reviewer.trim()) ? "--reviewer ${args.reviewer}" : ''
```

Comment:
Change of behaviour with some validation to avoid appending those flags when they are empty.

Otherwise:

```
[2021-07-01T13:52:21.282Z] + hub pull-request --push --message [automation] Update Elastic stack release version [7.13.2] --message ### What 

[2021-07-01T13:52:21.282Z]  Bump stack version with the latest release. 

[2021-07-01T13:52:21.282Z]  ### Further details 

[2021-07-01T13:52:21.282Z]  [7.13.2] --assign --reviewer --labels automation,dependency --base 7.x

[2021-07-01T13:52:24.594Z] remote: 

[2021-07-01T13:52:24.594Z] remote: Create a pull request for 'update-stack-version-20210701135218-7.x' on GitHub by visiting:        

[2021-07-01T13:52:24.594Z] remote:      https://github.com/elastic/apm-integration-testing/pull/new/update-stack-version-20210701135218-7.x        

[2021-07-01T13:52:24.594Z] remote: 

[2021-07-01T13:52:24.594Z] remote: GitHub found 8 vulnerabilities on elastic/apm-integration-testing's default branch (3 high, 5 moderate). To find out more, visit:        

[2021-07-01T13:52:24.594Z] remote:      https://github.com/elastic/apm-integration-testing/security/dependabot        

[2021-07-01T13:52:24.594Z] remote: 

[2021-07-01T13:52:24.594Z] To https://github.com/elastic/apm-integration-testing.git

[2021-07-01T13:52:24.594Z]  * [new branch]      HEAD -> update-stack-version-20210701135218-7.x

[2021-07-01T13:52:25.167Z] Error updating issue: Unprocessable Entity (HTTP 422)

[2021-07-01T13:52:25.167Z] Invalid value for "assignees"

[2021-07-01T13:52:25.179Z] Branch 'update-stack-version-20210701135218-7.x' set up to track remote branch 'update-stack-version-20210701135218-7.x' from 'origin'.

[2021-07-01T13:52:25.610Z] Terminated

githubCreatePullRequest: error hudson.AbortException: script returned exit code 1

```